cmd/run/run.go

```diff
@@ -214,6 +214,12 @@ type Config struct {
 	// ResolveNodeLimit indicates how deeply nested an authorization model can be.
 	ResolveNodeLimit uint32
 
+	// AllowWriting1Dot0Models allows writing of model with schema 1.0
+	AllowWriting1Dot0Models bool
```

Comment:
If we want to be consistent with other usages of 1.0 vs 1.1 we could use just `AllowWriting1_0Models`. We use `1_0` and `1_1` in our Go type names when referring to those.